src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Get-AzsDiskMigrationJob.ps1

```diff
@@ -0,0 +1,173 @@
+     
+    $GlobalParameterHashtable['SubscriptionId'] = $null
+    if($PSBoundParameters.ContainsKey('SubscriptionId')) {
+        $GlobalParameterHashtable['SubscriptionId'] = $PSBoundParameters['SubscriptionId']
+    }
+
+    $ComputeAdminClient = New-ServiceClient @NewServiceClient_params
+
+    $MigrationId = $Name
+
+ 
+    if('InputObject_DiskMigrationJobs_Get' -eq $PsCmdlet.ParameterSetName -or 'ResourceId_DiskMigrationJobs_Get' -eq $PsCmdlet.ParameterSetName) {
+        $GetArmResourceIdParameterValue_params = @{
+            IdTemplate = '/subscriptions/{subscriptionId}/providers/Microsoft.Compute.Admin/locations/{location}/diskmigrationjobs/{migrationId}'
+        }
+
+        if('ResourceId_DiskMigrationJobs_Get' -eq $PsCmdlet.ParameterSetName) {
+            $GetArmResourceIdParameterValue_params['Id'] = $ResourceId
+        }
+        else {
+            $GetArmResourceIdParameterValue_params['Id'] = $InputObject.Id
+        }
+        $ArmResourceIdParameterValues = Get-ArmResourceIdParameterValue @GetArmResourceIdParameterValue_params
+        $location = $ArmResourceIdParameterValues['location']
+
+        $migrationId = $ArmResourceIdParameterValues['migrationId']
+    }elseif (-not $PSBoundParameters.ContainsKey('Location')) {
+            $Location = (Get-AzureRMLocation).Location
+    }
+
+$filterInfos = @(
+@{
+    'Type' = 'powershellWildcard'
+    'Value' = $MigrationId
+    'Property' = 'Name' 
+})
+$applicableFilters = Get-ApplicableFilters -Filters $filterInfos
+if ($applicableFilters | Where-Object { $_.Strict }) {
+    Write-Verbose -Message 'Performing server-side call ''Get-AzsDiskMigrationJob -'''
+    $serverSideCall_params = @{
+
+}
+
+$serverSideResults = Get-AzsDiskMigrationJob @serverSideCall_params
+foreach ($serverSideResult in $serverSideResults) {
+    $valid = $true
+    foreach ($applicableFilter in $applicableFilters) {
+        if (-not (Test-FilteredResult -Result $serverSideResult -Filter $applicableFilter.Filter)) {
+            $valid = $false
+            break
+        }
+    }
+
+    if ($valid) {
+        $serverSideResult
+    }
+}
+return
+}
+    if ('DiskMigrationJobs_List' -eq $PsCmdlet.ParameterSetName) {
+        Write-Verbose -Message 'Performing operation ListWithHttpMessagesAsync on $ComputeAdminClient.'
+        $TaskResult = $ComputeAdminClient.DiskMigrationJobs.ListWithHttpMessagesAsync($Location, $(if ($PSBoundParameters.ContainsKey('Status')) { $Status } else { [NullString]::Value }))
+    } elseif ('DiskMigrationJobs_Get' -eq $PsCmdlet.ParameterSetName -or 'InputObject_DiskMigrationJobs_Get' -eq $PsCmdlet.ParameterSetName -or 'ResourceId_DiskMigrationJobs_Get' -eq $PsCmdlet.ParameterSetName) {
+        Write-Verbose -Message 'Performing operation GetWithHttpMessagesAsync on $ComputeAdminClient.'
+        $TaskResult = $ComputeAdminClient.DiskMigrationJobs.GetWithHttpMessagesAsync($Location, $MigrationId)
```

Comment:
Change to $MigrationId to $Name

Reply:
We'd keep what it is look like because it is autogerated. And I don't think it has issue or has any user experience effect.